mesonbuild/compilers.py

```diff
@@ -2263,6 +2267,34 @@ def get_gcc_soname_args(gcc_type, prefix, shlib_name, suffix, path, soversion, i
     else:
         raise RuntimeError('Not implemented yet.')
 
+def get_compiler_is_linuxlike(compiler):
+    if (getattr(compiler, 'gcc_type', None) == GCC_STANDARD) or \
+       (getattr(compiler, 'clang_type', None) == CLANG_STANDARD) or \
+       (getattr(compiler, 'icc_type', None) == ICC_STANDARD):
+           return True
```

Comment:
__[flake8]__
* [E111] indentation is not a multiple of four

<!-- Comment by [SideCI](https://sideci.com) -->